generators/app/templates/tsconfig.json

```diff
@@ -20,6 +20,10 @@
     "baseUrl": "src",
     "paths": {
       "@app/*": ["app/*"],
+      "@i18n": ["app/i18n"],
```

Comment:
As I commented in [your proposal here](https://github.com/ngx-rocket/generator-ngx-rocket/issues/277), I'm ok with adding these as a flag option, but not force it by default.
Anyways, that should be done in a separate PR as it's not directly related to i18n work here, could you change it back for this PR?

Reply:
I put these "third-party" modules (which could be reused) across the application here.

Reply:
Yes I understand the idea behind it, but it should not be forced by default as it's not supported properly by some IDEs and online editors. I'm for keeping this under an optional flag, but that should be done in a separate PR.